src/bin/capsule.rs

```diff
@@ -60,6 +60,19 @@ fn group_contracts_by_type(contracts: Vec<Contract>) -> HashMap<TemplateType, Ve
     contracts_by_type
 }
 
+fn get_last_args() -> (Vec<String>, Option<Vec<String>>) {
+    let args: Vec<String> = env::args().collect();
+    let mut iter = args.split(|n| n == "--");
```

Comment:
`split` can not handle situation like this `arg1 arg2 -- l2-arg1 l2-arg2 -- l3-arg1 l3-arg2`, use `splitn(2, ...)` instead.